### ISSUE TITLE
WCAG 1.1.1: Images must have alternate text (.styles_icon__lZfWw)

### ISSUE THREAD

**Issue by SahilTara** [Microsoft/WebTemplateStudio#121]
**Issue**: `Images must have alternate text` ([`image-alt`](https://dequeuniversity.com/rules/axe/3.1/image-alt?application=axeAPI))

**Target application**: [Project Acorn](http://localhost:3000/#/SelectWebApp)
**All Icons do this**
**Element path**: .styles_icon__lZfWw

**Snippet**:

    <img src="./static/media/fullstackwhite.4a51bb13.svg" class="styles_icon__lZfWw">

**How to fix**:

    Fix any of the following:
      Element does not have an alt attribute
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute or the title attribute is empty
      Element's default semantics were not overridden with role="presentation"
      Element's default semantics were not overridden with role="none"

**Environment**:
Chrome version 72.0.3626.121

====

This accessibility issue was found using Accessibility Insights for Web 2.0.4 (axe-core 3.1.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.
